test/test-condvar.c

```diff
@@ -235,7 +235,7 @@ TEST_IMPL(condvar_5) {
   uint64_t elapsed;
   uint64_t timeout;
 
-  timeout = 100 * 1e6; /* 100 ms in ns */
+  timeout = 100e6; /* 100 ms in ns */
```

Comment:
`100e6` is still a double literal.
```suggestion
  timeout = 100 * 1000 * 1000; /* 100ms in ns */
```